views/default/resources/favicon.ico.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * Handle requests for /favicon.ico
+ */
+
+elgg_set_http_header("HTTP/1.1 404 Not Found", true);
```

Comment:
Odd that is sets 404

Reply:
That was left over from when we used to send a 404. I'll remove it in my rebase of this.